cmd/helm/inspect.go

```diff
@@ -62,7 +69,41 @@ func newInspectCmd(c helm.Interface, out io.Writer) *cobra.Command {
 			return insp.run()
 		},
 	}
-	return cc
+
+	valuesSubCmd := &cobra.Command{
+		Use:   "values",
+		Short: "shows inspect values",
+		Long:  inspectDesc,
```

Comment:
My preference would be to have a unique `Long` description for each subcommand. Something like `inspectValuesDesc` and `inspectChartDesc`. As it is now, it was a little confusing when I read the help text:

```
⇒  helm inspect chart -h
This command inspects a chart (directory, file, or URL) and displays information.

Inspect prints the contents of the Chart.yaml file and the values.yaml file.

Usage:
  helm inspect chart [flags]

Global Flags:
      --debug         enable verbose output
      --home string   location of your Helm config. Overrides $HELM_HOME. (default "/Users/mattbutcher/Code/helm_home")
      --host string   address of tiller. Overrides $HELM_HOST. (default "localhost:44134")
```